psa_car_controller/psa/RemoteClient.py

```diff
@@ -83,6 +84,7 @@ def _on_mqtt_message(self, client, userdata, msg):  # pylint: disable=unused-arg
             elif msg.topic.startswith(MQTT_EVENT_TOPIC):
                 charge_info = data["charging_state"]
                 programs = data["precond_state"].get("programs", None)
+                self.doors_state[data["vin"]] = data["doors_state"]
```

Comment:
I don't know if doors_state is present for every one:
```suggestion
                if "door_state" in data:
                    self.doors_state[data["vin"]] = data["doors_state"]
```

Reply:
you're right! Thank you